src/fake-jni/jvm/string.cpp

```diff
@@ -26,7 +26,7 @@ namespace FakeJni {
   memcpy(array, str, slen);
  }
 
- bool JString::operator==(const JString& str) {
+ bool JString::operator==(const JString& str) const {
   return strcmp((char *)array, (char *)str.array) == 0;
```

Comment:
Should this maybe be a `memcmp`? Are the strings guaranteed to be valid (i.e.: NUL terminated)?

Reply:
Fixed in [04384ac](https://github.com/meme/fake-jni/commit/04384ace79e51170350856c43bde447542a7365a)